CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## [1.7.0] 2024-08-31
```

Comment:
@Elfi0Kuhndorf / @fernando-cortez do you want to add something about the DA sample here?

Reply:
Fernando answered with "nope" on slack, resolving